visualization/app/codeCharta/ui/codeMap/codeMap.preRender.service.spec.ts

```diff
@@ -201,5 +222,14 @@ describe("codeMapPreRenderService", () => {
 
 			expect(allNodesToBeExcluded()).toBeTruthy()
 		})
+
+		it("should change map to multiple mode and check that no id exists twice", () => {
+			const files = getCCFiles(storeService.getState().files)
+			storeService.dispatch(setMultiple(files))
```

Comment:
inline this, since files is only used once

```suggestion
			storeService.dispatch(setMultiple(getCCFiles(storeService.getState().files)))
```